src/main/java/org/apache/commons/text/StringTokenizer.java

```diff
@@ -130,10 +130,12 @@
     private StringMatcher trimmerMatcher = StringMatcherFactory.INSTANCE.noneMatcher();
 
     /** Whether to return empty tokens as null. */
-    private boolean emptyAsNull = false;
+    private boolean emptyAsNull;
     /** Whether to ignore empty tokens. */
     private boolean ignoreEmptyTokens = true;
 
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```

Comment:
-1, reuse ArrayUtils.

Reply:
Changed  for ArrayUtils.EMPTY_STRING_ARRAY